src/app/recruit/[id]/edit/panel/SetTeamRole/BasicSelectMember.tsx

```diff
@@ -22,9 +25,15 @@ export default function BasicSelect({
           id="demo-simple-select"
           value={member}
           onChange={handleChange}
+          sx={{ width: '4.5rem', height: '2rem' }}
+          disabled={disabled ? disabled : false}
```

Comment:
disabled가 이미 boolean으로 처리되어 있는데, 굳이 삼항연산자로 표기할 필요 없어보입니다. 문제될 것은 아니지만 코멘트 남겨둬요
```suggestion
          disabled={disabled}
```